src/Pim/Bundle/CatalogBundle/Doctrine/ORM/CompletenessRemover.php

```diff
@@ -56,13 +56,16 @@ public function __construct(
      */
     public function removeForProduct(ProductInterface $product)
     {
-        $product->getCompletenesses()->clear();
-        $statement = $this->entityManager->getConnection()->executeQuery(
-            'DELETE c FROM pim_catalog_completeness c JOIN pim_catalog_product p WHERE p.identifier = ?',
```

Comment:
Because we try to use identifier anywhere possible, but in SQL, we should use ID when simpler.